execute/factory.go

```diff
@@ -135,6 +136,15 @@ func (p PluginFactory) NewReportingPlugin(
 		return nil, ocr3types.ReportingPluginInfo{}, fmt.Errorf("failed to create token data observer: %w", err)
 	}
 
+	backgroundTokenDataObserver := tokendata.NewBackgroundObserver(
+		logger.Named(p.lggr, "BackgroundObserver"),
+		baseObserver,
+		10,             /* numWorkers */
+		10*time.Minute, /* cacheExpirationInterval */
+		15*time.Minute, /* cacheCleanupInterval */
+		5*time.Second,  /* observeTimeout */
```

Comment:
Similarly to the previous comment, probably these values are going to be specific to the integration, no?